test/functional/cmdLineTests/valuetypeddrtests/flattenedvaluetypeddrtests.xml

```diff
@@ -157,9 +158,9 @@
 			<input>quit</input>
 		</command>
 		<output regex="no" type="success" showMatch="yes"> ValueTypeWithVolatileFields</output>
-		<output regex="no" type="required" showMatch="yes"> point (offset = 4) (Point2D)</output>
-		<output regex="no" type="required" showMatch="yes"> vpoint (offset = 12) (Point2D)</output>
-		<output regex="no" type="required" showMatch="yes"> line (offset = 20) (FlattenedLine2D)</output>
+		<output regex="no" type="required" showMatch="yes"> point (offset = 20) (Point2D)</output>
```

Comment:
what is the full layout of this type? is it?

```
class ValueTypeWithVolatileFields {
    ValueInt i; //offset 0 ??
    volatile Point2D vpoint; //offset 4
    ValueInt i2; //offset 16 ??
    Point2D point; //offset 20
    Line2D line; //offset 28
    volatile Line2D vline //offset 44?
}
```

Reply:
It is:

```
!J9Object 0x00000000FFEC5508 {
        struct J9Class* clazz = !j9class 0x1BC300 // ValueTypeWithVolatileFields
        Object flags = 0x00000000;
        QValueInt; i = !j9object 0xffec5508 i (offset = 12) (ValueInt)
        QValueInt; i2 = !j9object 0xffec5508 i2 (offset = 16) (ValueInt)
        QPoint2D; point = !j9object 0xffec5508 point (offset = 20) (Point2D)
        QPoint2D; vpoint = !j9object 0xffec5508 vpoint (offset = 4) (Point2D)
        QFlattenedLine2D; line = !j9object 0xffec5508 line (offset = 28) (FlattenedLine2D)
        QFlattenedLine2D; vline = !fj9object 0xffec5400 (offset = 0) (ValueTypeWithVolatileFields)
}
```